docs/install-upgrade/config.md

```diff
@@ -146,50 +146,91 @@ Switches push telemetry data through a proxy running in a pod on the control
-can be pushed to a Grafana instance at the customer environment, or to Grafana
-cloud.
 
-```{ .yaml title="telemetry.yaml" linenums="1" }
+#### Credentials
+
+Telemetry can be enabled after installation of the fabric. There are two YAML
+objects that control the telemetry configuration. The first object provides the
+location and credentials for sending the telemetry.
+
+```{ .yaml .annotate title="credentials.yaml" linenums="1" }
 spec:
   config:
-    fabric:
-      defaultAlloyConfig:
-        agentScrapeIntervalSeconds: 120
-        unixScrapeIntervalSeconds: 120
-        unixExporterEnabled: true
-        lokiTargets:
-          grafana_cloud: # target name, multiple targets can be configured
-              basicAuth: # optional
-                  password: "<password>"
-                  username: "<username>"
-              labels: # labels to be added to all logs
-                  env: env-1
-              url: https://logs-prod-021.grafana.net/loki/api/v1/push
-              useControlProxy: true
-        prometheusTargets:
-          grafana_cloud: # target name, multiple targets can be configured
-              basicAuth: # optional
-                  password: "<password>"
-                  username: "<username>"
-              labels: # labels to be added to all metrics
-                  env: env-1
-              sendIntervalSeconds: 120
-              url: https://prometheus-prod-36-prod-us-west-0.grafana.net/api/prom/push
-              useControlProxy: true
-        unixExporterCollectors: # list of node-exporter collectors to enable, https://grafana.com/docs/alloy/latest/reference/components/prometheus.exporter.unix/#collectors-list
-        - cpu
-        - filesystem
-        - loadavg
-        - meminfo
-        collectSyslogEnabled: true # collect /var/log/syslog on switches and forward to the lokiTargets
+    observability:
+      targets:
+        loki:
+          grafana_cloud: # (1)!
+            basicAuth:
+              password: "insert_password_or_token_here"
+              username: "username"
+            labels: 
+              env: hh-fabric # (2)!
+            url: https://[your-loki-server].grafana.net/loki/api/v1/push
+        prometheus:
+          grafana_cloud: # (1)!
+            basicAuth:
+              password: "insert_password_or_token_here"
+              username: "username"
+            labels:
+              env: hh-fabric
+            url: https://[your-prometheus-server].grafana.net/api/prom/push
+
 ```
 
+1. Can be any name of your choosing
+2. Change to match your environment
+
 To enable the telemetry after install use:
 
 ``` shell
-kubectl patch -n fab --type merge fabricator/default --patch-file telemetry.yaml
+kubectl patch -n fab --type merge fabricator/default --patch-file credentials.yaml
+```
+
+
+#### Collecting and Pushing
+
+By default the full list of telemetry is sent from the fabric to Prometheus and 
+Loki. Alloy allows for 
```

Comment:
let's finish the sentence here (I assume something about filtering metrics using regexes and/or relabelling them). I would also explicitly say if the collectors listed here are exhaustive. Maybe some useful regexes beyond the one we include in the sample, if it makes sense to add them?